game/scripts/vscripts/libraries/timers.lua

```diff
@@ -191,7 +196,8 @@ function Timers:HandleEventError(name, event, err)
 	end
 end
 
-function Timers:CreateTimer(name, args, context)
+function Timers:CreateTimer(name, args, context, countTicks)
```

Comment:
Instead of adding a new argument use a property of `args` (which you already assign below)